docs/src/componentDocs/ToolbarMenu/playground/PreviewComponent.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { RootState } from '../../../redux/store';
+import { useAppSelector } from '../../../redux/hooks';
+import { createProps, getIcon, removeEmptyLines } from '../../../shared/utilities';
+import { PropsType } from '../../../__types__';
+import PreviewComponentWithCode from '../../../shared/PreviewComponentWithCode';
+import { ToolbarMenu } from '@brightlayer-ui/react-components/core/ToolbarMenu';
+
+import * as Colors from '@brightlayer-ui/colors';
+
+export const PreviewComponent = (): JSX.Element => {
+    const toolbarMenuJson = useAppSelector((state: RootState) => state.componentsPropsState.toolbarMenuComponent);
+
+    const toolbarMenuProps = createProps(toolbarMenuJson.props as PropsType[]);
+
+    const iterateIconProps = (iconProps: any): string => {
```

Comment:
there are a lot of `any` types. Is there something like `IconProps` exported from MUI?

Reply:
> there are a lot of `any` types. Is there something like `IconProps` exported from MUI?

Moved this to utlities.tsx